daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_functions/Functions.tdml

```diff
@@ -6805,7 +6812,8 @@
     <xs:element name="dfdlULong09" type="xs:unsignedLong" dfdl:inputValueCalc="{ dfdl:unsignedLong(dfdl:hexBinary(9223372036854775807)) }"/> <!-- valid -->
     <xs:element name="dfdlULong11" type="xs:unsignedLong" dfdl:inputValueCalc="{ dfdl:unsignedLong('xFFFFFFFFFFFFFFFG') }"/> <!-- invalid -->
     <xs:element name="dfdlULong12" type="xs:unsignedLong" dfdl:inputValueCalc="{ dfdl:unsignedLong(-1) }"/> <!-- invalid -->
-    
+    <xs:element name="dfdlULong13" type="xs:unsignedLong" dfdl:inputValueCalc="{ dfdl:unsignedLong('  18446744073709551615') }"/> <!-- valid -->
```

Comment:
The JIRA issue says XPath performs whitespace normalization which removes leading AND trailing whitespace.  None of your new tests have any trailing whitespace, even though the trim method will remove it too.  Other than that, I'll let Mike and Steve review.